cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -235,6 +235,26 @@ def test_monthly_charges(appliance, has_no_providers_modscope, setup_provider, c
         assert monthly_charges != '$0'
 
 
+@pytest.mark.long_running
+@pytest.mark.parametrize('context', [ViaSSUI])
+@pytest.mark.parametrize('order_service', ['vm_count_1', 'vm_count_2'], indirect=True)
```

Comment:
this doesn't look optimal. below is example how you can implement this:

```
import pytest


@pytest.fixture
def my_lovely_fixture(request):
    if isinstance(request.param, list):
        print(request.param[0], request.param[1])
    if isinstance(request.param, dict):
        print(request.param)
    yield request.param


@pytest.mark.parametrize('my_lovely_fixture', [{'console_test': True, 'vm_count': 1},
                                               {'console_test': False, 'vm_count': 2}],
                         indirect=True, ids=["vm-count1", "vm-count2"])
def test_my_lovely_test(my_lovely_fixture):
    assert True


@pytest.mark.parametrize('my_lovely_fixture', [[True, 1], [False, 2]],
                         indirect=True, ids=["vm-count1", "vm-count2"])
def test_my_lovely_test2(my_lovely_fixture):
    assert True
```

output:
```
test_params.py::test_my_lovely_test[vm-count1] {'console_test': True, 'vm_count': 1}
PASSED                    [ 25%]
test_params.py::test_my_lovely_test[vm-count2] {'console_test': False, 'vm_count': 2}
PASSED                    [ 50%]
test_params.py::test_my_lovely_test2[vm-count1] True 1
PASSED                   [ 75%]
test_params.py::test_my_lovely_test2[vm-count2] False 2
PASSED                   [100%]

============================== 4 passed in 0.02s ===============================
```